docker-compose.override.yml

```diff
@@ -19,3 +19,9 @@ services:
     environment:
       CERYX_API_HOSTNAME: ${CERYX_API_HOSTNAME:-api.ceryx.dev}
       CERYX_DEBUG: ${CERYX_DEBUG:-true}
+
+networks:
+  default:
+    attachable: true

```

Comment:
This is not required, `attachable` makes sense only for `overlay` networks.

Reply:
Right! Removed in https://github.com/sourcelair/ceryx/pull/42/commits/f3400ffc9a34547ff5e0d5052259a298b150a1ee.